backend/services/cloudkit_web_client.py

```
@@ -0,0 +1,326 @@
+        if not self.enabled:
+            raise Exception("CloudKit Web Services not properly configured")
+        
+        # Use provided ISO date or generate new one
+        if not iso_date:
+            iso_date = datetime.utcnow().strftime('%Y-%m-%dT%H:%M:%SZ')
+        
+        # Calculate body hash (SHA-256 then base64)
+        if data:
+            request_body = json.dumps(data, separators=(',', ':'))
+        else:
+            request_body = ""
```

Ruff (0.11.9)

Use ternary operator `request_body = json.dumps(data, separators=(',', ':')) if data else ""` instead of `if`-`else`-block

Replace `if`-`else`-block with `request_body = json.dumps(data, separators=(',', ':')) if data else ""`

(SIM108)